log4tango/config/config.cmake

```diff
@@ -56,6 +56,10 @@ LOG4TANGO_CHECK_COMPILER_FEATURE("config/check_namespace.cpp" NAMESPACES)
 #snprintf
 LOG4TANGO_CHECK_COMPILER_FEATURE("config/check_snprintf.cpp" SNPRINTF)
 
+if(NOT LOG4TANGO_HAVE_SNPRINTF)
+  message(FATAL_ERROR "snprintf could not be found, aborting.")
+endif()
```

Comment:
Do we still need this check if presence of snprinf is guaranteed by the C++ standard required by tango (CXX_STANDARD_REQUIRED)?

Reply:
You are right, let's remove the check for `snprintf`.

Reply:
@mliszcz Updated.